docker-compose.yml

```diff
@@ -31,7 +31,7 @@ services:
           # Ideally this stack name would be scoped by organization, but
           # login --local doesn't support organizations
           pulumi stack init local-grapl-integration-tests --non-interactive
-          pulumi up --yes --skip-preview --stack=local-grapl-integration-tests
+          pulumi up --yes --skip-preview --stack=local-grapl-integration-tests --tracing http://host.docker.internal:9411/api/v1/spans
```

Comment:
Since this command is basically one big chunk of bash script, can we set this URL as a variable up at the top and just reference it in both these places?